packages/shared/test-utils/src/mocks.ts

```
@@ -0,0 +1,116 @@
+    model: 'gpt-4-turbo-preview',
+    usage: {
+      completionTokens: 50,
+      promptTokens: 100,
+      totalTokens: 150,
+    },
+  }
+}
+
+// Mock WebSocket connection
+export function createMockSocket() {
+  const events = new Map<string, Function[]>()
```

ESLint

[error] The `Function` type accepts any function-like value.
Prefer explicitly defining any function parameters and return type.

(@typescript-eslint/no-unsafe-function-type)